scripts/west_commands/runners/qemu.py

```diff
@@ -4,28 +4,210 @@
 
 '''Runner stub for QEMU.'''
 
-from runners.core import ZephyrBinaryRunner, RunnerCaps
+import argparse
+import logging
+import os
+from pathlib import Path
+import shlex
+import subprocess
+import sys
+import yaml
 
+from runners.core import ZephyrBinaryRunner, RunnerCaps, FileType
+
+DEFAULT_QEMU_EXE = 'qemu-system-arm_rt595.exe' if sys.platform == 'win32' else 'qemu-system-arm_rt595'
```

Comment:
Why? It seems like there should be no default, and the board.cmake file should always specify. Am I missing something?

Reply:
right, for now there should be using default, this is my test qemu, rt595 is not in zephyr-sdk qemu support list yet. let me update them

Reply:
The current PR changes the default, but my original comment was "It seems like there should be no default"; why was this marked resolved?